data/brands/amenity/bank.json

```
@@ -7561,18 +7561,30 @@
       }
     },
     {
-      "displayName": "State Employees Credit Union",
+      "displayName": "State Employees Credit Union (North Carolina)",
       "id": "stateemployeescreditunion-ea2e2d",
       "locationSet": {"include": ["us"]},
       "tags": {
         "amenity": "bank",
         "brand": "State Employees Credit Union",
         "brand:wikidata": "Q7603196",
-        "brand:wikipedia": "en:State Employees Credit Union",
+        "brand:wikipedia": "en:State Employees Credit Union (North Carolina)",
         "name": "State Employees Credit Union",
         "short_name": "SECU"
       }
     },
+    {
+      "displayName": "State Employees Credit Union (Maryland)",
+      "locationSet": {"include": ["us"]},
+      "tags": {
+        "amenity": "bank",
+        "brand": "State Employees Credit Union",
+        "brand:wikidata": "Q100386180",
+        "brand:wikipedia": "en:State Employees Credit Union of Maryland",
+        "name": "State Employees Credit Union",
+        "short_name": "SECU MD"
```

They just go by SECU on their branding

```suggestion
        "short_name": "SECU"
```